samples/graph-appcatalog-lifecycle/csharp/AppCatalogSample/Helper/OAuthHelpers.cs

```diff
@@ -0,0 +1,262 @@
+﻿using AdaptiveCards;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+using Microsoft.Graph;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace AppCatalogSample.Helper
+{
+    public static class OAuthHelpers
+    {
+        public static async Task<IList<TeamsApp>> GetAllapp(ITurnContext turnContext, TokenResponse tokenResponse)
+        {
+            try
+            {
+                var client = new AppCatalogHelper(tokenResponse.Token);
+                return await client.GetAllapp();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+                return null;
+            }
+
+        }
+
+        public static async Task<IList<TeamsApp>> AppCatalogById(ITurnContext turnContext, TokenResponse tokenResponse)
```

Comment:
Check if these additional wrapper methods are required. I believe you can directly call these instead.